client/src/components/RatingsNReviews/NewReview.jsx

```diff
@@ -59,17 +59,22 @@ const NewReview = ({setIsPost, setOpenReviewModal})=>{
     }});
   const [rating, setRating] = useState(null);
   const [hover, setHover] = useState(null);
+  const [reviewSummary, setReviewSummary] = useState('');
+  const [reviewBody, setReviewBody] = useState('');
 
   const postReview = function () {
-    axios.post('http://localhost:3000/ratings/postReview')
-      .then(()=>{
-        setIsPost(true);
-        setOpenReviewModal(false);
-      })
-      .catch((err) => {
-        console.log('client side postReview error:', err);
-      });
+    console.log('Do post review:', reviewSummary, reviewBody);
+    // axios.post('http://localhost:3000/ratings/postReview')
```

Comment:
Does POST request should be here or shouldn't?

Reply:
It should be here, I just comment it out for testing purposes